### DISCUSSION TITLE
Courseplay un IT Runner

### DISCUSSION THREAD

**Discussion by mikeatworkgp** [Courseplay/Courseplay_FS22#2433] [General]
Hallo ich habe ein Problem mit dem IT Runner. Wenn ich eine Anhänger mit it Runner Funktion  habe und CP Starte wird das obere Teil automatisch abgeladen.  Ich habe da Probleme mit den Düngerstreuer. Hier ein foto

Danke
![fs](https://user-images.githubusercontent.com/129971199/230339085-a7a64401-e3cd-4623-a740-ecc7abb475fa.jpg)

**Comment by Tensuko** [comment]
That is becouse the IT Runner have X as unfold and unfold allowed by AI.
Thats a mod Issue.
